src/platforms/dummy/dummypingsender.cpp

```diff
@@ -0,0 +1,24 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
It would be better to use a `git mv` to keep the history.

Reply:
I did, but I think with the name change from `xxxSendWorker` to `xxxSender` meant that too many lines had changed for git to be able to identify them as the same file.